active/radiobuttons/RadioButtonsAPI.md

```diff
@@ -0,0 +1,155 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a new control that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
```

Comment:
I think it is confusing that RadioButtons is control, but we are refering to collections of objects of the RadioButton class as RadioButtons in this spec as well. Perhaps we could be a little more verbose to avoid this confusion by replacing the RadioButtons which do not refer to the new control with "Groups of RadioButton objects",